packages/tools/fluid-runner/README.md

```diff
@@ -31,10 +31,36 @@ fluidRunner({ /* IFluidFileConverter implementation here */ });
 The input file is expected to be an ODSP snapshot.
 For some examples, see the files in the [localOdspSnapshots folder](./src/test/localOdspSnapshots).
 
+### Telemetry format
+There is an optional command line option `telemetryFormat` that allows you to specify the telemetry output format. The naming provided to this option is strict and must match an option in [OutputFormat](./src/logger/fileLogger.ts).
+The default format is currently `JSON`
+
+### Additional telemetry properties
+There is an optional command line option `telemetryProps` that allows you to specify additional properties that will be added to every telemetry entry. The format of this option is strict and must match the following:
+- every key is separated from the value by the `=` symbol
+- every key-value-pair is separated by whitespace
```

Comment:
why whitespace? I think comma or semicolon is usually used.

Reply:
I do not know if the library we use allows it, but I think the better format would be something like that:
--prop name value --prop name2 value2 ....
This way we are not limiting usage - all chars can be used, and users just need to understand how escaping works in the shell they are using (i.e., if they want to use quotation marks as payload)

Reply:
This is possible given the yargs implementation. Can do basically any combination of grouping as yargs has good built-in support for calling options multiple times.